src/App.js

```diff
@@ -1,16 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [messageData, setMessageData] = useState(chatMessages);
+
+  const calcTotalLikes = (messageData) => {
```

Comment:
Way to use `reduce` here! It's cool that you leveraged the knowledge that in JavaScript if you add a boolean to something it gets cast as a number (0 for false, 1 for true) although you probably would see most people use conditional logic to add a number to a number instead.